src/components/Search.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import SearchBar from './SearchBar';
+import SearchResults from './SearchResults';
+import Autocomplete from './Autocomplete';
+import foodList from '../foodList';
+
+const Search = () => {
+  const [input, setInput] = useState('');
+  const [filteredList, setFilteredList] = useState(null);
+  const [result, setResult] = useState('');
+  const [pointer, setPointer] = useState(-1);
+
+  const updateInput = (input) => {
+    const inputString = input.trim().toLowerCase();
+    const filtered = foodList.filter((food) => {
+      return food.name.toLowerCase().includes(inputString);
+    });
+    setInput(input);
+    setFilteredList(inputString === '' ? null : filtered.slice(0, 5));
```

Comment:
[minor] I'd suggest flipping this to make it less verbose:
```suggestion
    setFilteredList(inputString ? filtered.slice(0, 5) : null);
```